gradle-plugin/build.gradle

```diff
@@ -124,6 +133,7 @@ repositories {
 dependencies {
   implementation 'com.google.protobuf:protobuf-gradle-plugin:0.8.19'
   implementation 'commons-lang:commons-lang:2.6'
+  runtime 'com.google.guava:guava:32.1.2-android'
```

Comment:
why it's `android`?

Reply:
The io.grpc jars depend on the android variants of the guava jars.

https://mvnrepository.com/artifact/io.grpc/grpc-core/1.54.2

Presumably the android variants of the guava jars work well on android and on full Java runtimes.